dashboard/config/routes.rb

```diff
@@ -751,7 +755,9 @@
   get 'foorm/preview/:name', to: 'foorm_preview#name', constraints: {name: /.*/}
   get 'foorm/preview', to: 'foorm_preview#index'
 
-  get 'foorm/editor', to: 'foorm_editor#index', constraints: {name: /.*/}
+  get 'foorm/editor', to: 'foorm_form_editor#index'
+  get 'foorm/editor/forms', to: 'foorm_form_editor#index'
+  get 'foorm/editor/libraries', to: 'foorm_library_editor#index'
```

Comment:
Not sure on Rails/JS singular/plural choices, if anyone has opinions.

Reply:
Personally I like `foorm/form/editor` since we are only ever editing one form at a time and it makes more sense grammatically. But am not sure if that's the right convention.